Earthfile

```diff
@@ -40,7 +40,7 @@ build-crate:
 lint:
     FROM +copy-src
     RUN rustup component add clippy
-    RUN cargo clippy --all-features --all-targets -- -D warnings
+    RUN cargo clippy --all-features --all-targets -- -D warnings -D clippy::undocumented_unsafe_blocks
```

Comment:
Lets instead do this via `#![warn( clippy::undocumented_unsafe_blocks)]` at the top of `lib.rs`, so devs can just run `cargo clippy` .